### PR TITLE
Cookie rules

### PR DESCRIPTION
**Pull request by gork7777**
issue: #89460
issue: #89455
issue: #89448
issue: #89456

### CODE REVIEW

File: AnnoyancesFilter/sections/cookies_specific.txt [hunk 1]
```diff
@@ -5636,7 +5636,7 @@ arpaceramiche.it,firefox-usb.com,m64.info##.jb.cookie
 softpedia.com##.jobnotice_cnt
 yahoo.com##.js-applet.windowshade .fixed-win.Px-10.Bd-1
 chessable.com,leroymerlin.fr##.js-banner-cookie
-daimler-tss.com,gofundme.com##.js-cookie-banner
+made.com,##.cookiedaimler-tss.com,gofundme.com##.js-cookie-banner
```
Comment (Sergey-Lyapin) [inline AnnoyancesFilter/sections/cookies_specific.txt:5639]: `made.com,daimler-tss.com,gofundme.com##.js-cookie-banner`

**Review by Sergey-Lyapin — CHANGES_REQUESTED** [review]
@gork7777

**Comment by Sergey-Lyapin** [comment]
and for newsletter `made.com##div[data-newsletter-signup]`

File: AnnoyancesFilter/sections/cookies_specific.txt (hunk 1 above)
Comment (gork7777) [reply AnnoyancesFilter/sections/cookies_specific.txt:5639]: Oh I didn't see this, I will fix this

**Pull request merged**